tests/test_models/test_tbats.py

```diff
@@ -375,6 +394,7 @@ def test_arma_with_seasonal_components_not_fitted(periodic_dfs, estimator, metho
 
 
 @pytest.mark.filterwarnings("ignore:.*not fitted.*")
+@pytest.mark.parametrize("dfs_name", ["periodic_dfs", "periodic_dfs_int_timestamp"])
```

Comment:
I'm not sure it is a good place to check components for int timestamp.

Reply:
Here I check that components are built correctly if timestamp is integer. But I'm not sure that adding df with integer timestamp in exactly this test is a good idea. We could probably add this case into some other test that works with components prediction.

Reply:
I think, It's okey. What do you see as the problem?